cmd/eval_test.go

```diff
@@ -273,7 +273,36 @@ func testEvalWithInvalidSchemaFile(t *testing.T, input string, query string, sch
 	return err
 }
 
-func TestEvalWithJSONSchemaFile(t *testing.T) {
+func testReadParamWithSchemaDir(t *testing.T, input string, query string, inputSchema string) error {
```

Comment:
🔍 Since this is a test function, and we're passing `t *testing.T` already, we don't need the error return. We can call

```go
t.Fatalf("Unexpected error or undefined from evaluation: %v", err)
```
directly instead of returning the error.

Alternatively, we could drop the `testing.T` argument. But I think I'd rather drop the error return.

Reply:
The error return is nice to have for the `test*` functions so that the main test cases can handle whether or not the error was expected, and react accordingly. This is also consistent with the existing test case pattern in `eval_test.go` for example, this one https://github.com/open-policy-agent/opa/blob/d3fb38adff4c879a2fd9d6c3c7265e2a23f9c14e/cmd/eval_test.go#L120

I think it's better to leave that `testReadParamWithSchemaDir` as-is with the error return. Is that ok?

Reply:
Yeah it's fine. Just a test. :+1: